backend/src/telegram/models/telegram-subscriber.ts

```diff
@@ -8,9 +8,15 @@ import { Column, Model, Table, DataType } from 'sequelize-typescript'
 export class TelegramSubscriber extends Model<TelegramSubscriber> {
   @Column({
     type: DataType.STRING,
+    primaryKey: true,
+    unique: true,
```

Comment:
Not sure if we still need to set `unique` here given that primary keys are already unique.

Reply:
I added `unique` here because the primary key is compound. Considering this situation:

```
phone_1: telegram_1
phone_1: telegram_2
```

The primary key unique constraint would still allow this, but to us it's invalid. 

But now I'm re-thinking if we even need a compound primary key here. I think a possible alternative is to set one of the columns as the primary key and the other as unique - this indexes both columns anyway, so the compound primary key just seems like an extra index for no benefit?